wsmaster/che-core-api-factory-bitbucket/src/main/java/org/eclipse/che/api/factory/server/bitbucket/BitbucketApiClient.java

```diff
@@ -228,6 +235,8 @@ private <T> T executeRequest(
             throw new ScmBadRequestException(body);
           case HTTP_NOT_FOUND:
             throw new ScmItemNotFoundException(body);
+          case HTTP_UNAUTHORIZED:
+            throw new ScmUnauthorizedException(body, "github", "v1", "");
```

Comment:
Can we move constant somewhere else? 
Copy/paste from github?

Reply:
>Copy/paste from github?

right, fixed
>Can we move constant somewhere else?

Could you please elaborate on this?